lib/jekyll-feed/feed.xml

```diff
@@ -11,20 +11,20 @@
 
   {% assign title = site.title | default: site.name %}
   {% if page.collection != "posts" %}
-    {% assign collection = page.collection | capitalize %}
-    {% assign title = title | append: " | " | append: collection %}
+    {% assign title = page.title | default: site.name %}
```

Comment:
Lots of overlapping and redundancy here..
What is your expected output in a site with **just the following YAML configuration**:
```yaml
title: "Hello World"
collections:
  puppies:
    output: true
plugins: ["jekyll-feed"]
```

Reply:
I've added a commit that makes this part of the `feed.xml` template a bit clearer. But the tests already account for a YAML configuration like this.

The expected `<title>` output here would be:

```
Hello World | puppies
```